Header header
float64[] ranges
bool[] los
uint32[] ids
uint32[] rangeIndx
bool valid
uint32 codec
uint32 maxUwbNo
